tests/ChatTests/Groups.hs

```diff
@@ -3047,3 +3048,65 @@ testMemberContactIncognito =
         [ alice <# ("#team " <> cathIncognito <> "> hey"),
           bob ?<# ("#team " <> cathIncognito <> "> hey")
         ]
+
+testMemberContactProfileUpdate :: HasCallStack => FilePath -> IO ()
+testMemberContactProfileUpdate =
+  testChat3 aliceProfile bobProfile cathProfile $
+    \alice bob cath -> do
+      connectUsers alice bob
+      createGroup2 "team" bob cath
+      bob ##> "/a #team alice"
+      bob <## "invitation to join the group #team sent to alice"
+      alice <## "#team: bob invites you to join the group as member"
+      alice <## "use /j team to accept"
+      alice ##> "/j team"
+      concurrentlyN_
+        [ do
+            alice <## "#team: you joined the group"
+            alice <## "#team: member cath (Catherine) is connected",
+          do
+            bob <## "#team: alice joined the group",
+          do
+            cath <## "#team: bob added alice (Alice) to the group (connecting...)"
+            cath <## "#team: new member alice is connected"
+        ]
+      alice #> "#team hello"
+      bob <# "#team alice> hello"
+      cath <# "#team alice> hello"
+      cath #> "#team hello too"
+      bob <# "#team cath> hello too"
+      alice <# "#team cath> hello too"
+
+      alice ##> "/p alisa Alisa"
+      alice <## "user profile is changed to alisa (Alisa) (your 1 contacts are notified)"
+      bob <## "contact alice changed to alisa (Alisa)"
+      bob <## "use @alisa <message> to send messages"
+
+      alice `send` "@cath hello"
+      -- TODO too many outputs here, and maybe should follow the output of the message?
+      -- two lines - connecting..., connected would be ok
+      alice <## "member #team cath does not have direct connection, creating"
```

Comment:
this output is only in CLI so that if there's error on creating contact, it's more clear what chat was trying to do. For example, there're tests where there's failure immediately due to version incompatibility and w/t this line it looks like:

"@cath hello"
"peer chat version range incompatible"

with this line it's at least clear chat was trying to create member contact